src/task10/task10.js

```diff
@@ -0,0 +1,21 @@
+var obj = JSON.parse(ANCESTRY_FILE)
+
+function average(array) {
```

Comment:
If you start to short, try this:
`const average = array => array.reduce((a, b) => a + b) / array.length;`